### ISSUE TITLE
[Docs] Fix update-docs to not delete bucket files and run only  after_deploy

### ISSUE THREAD

**Issue by pelsasser** [MARKETProtocol/MARKET.js#164]
## Before you `start work`

Please read our contribution [guidelines](https://docs.marketprotocol.io/#contributing) and if there is a bounty involved  please also see [here](https://docs.marketprotocol.io/#gitcoin-and-bounties)

If you have ongoing work from other bounties with us where funding has not been released, please do not pick up a new issue.  We would like to involve as many contributors as possible and parallelize the work flow as much as possible.

Please make sure to comment in the issue here immediately after starting work so we know your plans for implementation and a timeline.

Please also note that in order for work to be accepted, all code must be accompanied by test cases as well.

### Why Is this Needed?
*Summary*: Currently, when the update-docs script runs it deletes all files in the bucket that aren't locally there. So our marketprotocol/docs files for docs.marketprotocol.io get deleted as well.  Also, we should only run this script on after_deploy, no need to run this script for every build

### Description
*Type*:  Bug

### Current Behavior
Scripts run after every build and blows away anything but the market.js docs in the bucket

### Expected Behavior
Script runs only after a tagged master commited (release) and doesn't remove all the files in the bucket that we need. 

### Reproduction
disabled due to docs.marketprotocol.io being a production bucket

### Solution
*Summary*: Fix script and .travis.yml

### Definition of Done
- [ ] update `tools/update-docs.js` to not delete the non local files from the bucket
- [ ] modify `tools/update-docs.js` to upload to a bucket sub directory called /marketjs/ and leave anything in the root directory untouched.
- [ ] modify .travis.yml to only run this script `after_deploy`
